drawBot/context/tools/openType.py

```diff
@@ -485,6 +488,7 @@ def getFeatureTagsForFontAttributes(attributes):
     return featureTags
 
 
+@memoize
 def getFeatureTagsForDescriptions(featureDescriptions):
```

Comment:
This one only gets called by `getFeatureTagsForFontName()`, which is already being memoized. So memoizing this one doesn't make sense.

Reply:
Also, the `featureDescriptions` object comes from CoreText, should be a CFArrayRef, and I doubt it's hashable in a meaningful way (it may compare and hash by identity, which makes it unsuitable as a cache key).